nodejs/eks/examples/examples_test.go

```diff
@@ -39,6 +39,7 @@ func TestAccCluster(t *testing.T) {
 					info.Deployment.Resources,
 					info.Outputs["kubeconfig1"],
 					info.Outputs["kubeconfig2"],
+					info.Outputs["kubeconfig3"],
```

Comment:
We're going to have to rethink how we test for this example in the k8s test framework, as we currently parse the stack resources searching for nodes to compute a `desiredNodeCount` to anticipate during testing after a cluster has come up:

https://github.com/pulumi/pulumi-eks/blob/master/utils/utils.go#L487

Reply:
I added a change that skips validating the number of Nodes that exist if there are no NodeGroups attached.  I am hopeful that unblocks this, though we may need more flexibility in the test framework for future tests.